Sage/carbon
# feat(select): make select list respond to pageup/pagedown keys

### Proposed behaviour



When focused on a Select textbox and pressing the Page Up or Page Down keys (on keyboards lacking these specific keys, the same functionality can be triggered by pressing the up/down arrow while holding the Function key), the list of options should scroll up/down. Also when using Voice Control, the "scroll up" and "scroll down" commands should have the same effect.

### Current behaviour

Pressing Page Up/Down, or giving the Voice Control command scroll up/down, does nothing.



### Checklist




#### QA


### Additional context



### Testing instructions

Test each type/variation of Select and ensure that sensible behaviour is present for each of the following:
- pressing the pageup/pagedown key manually (Function+Up/Down)
- using Voice Control on Mac and giving the voice command "scroll up" or "scroll down"



The following CodeSandbox is an example of the broken behaviour.
You can see the new behaviour by looking at the version in the comment by `codesandbox[bot]`.
